test/modules/core/lib/attribute-manager.spec.js

```diff
@@ -176,7 +176,7 @@ test('AttributeManager.update - external buffers', t => {
     }
   });
 
-  t.is(attribute.buffer.accessor.type, gl.UNSIGNED_BYTE, 'colors casted to correct type');
+  t.is(attribute.type, gl.FLOAT, 'colors accessor is set to correct type');
```

Comment:
A test case setting positions to UNSIGNED_INT would be perfect, but I gather this comes in next PR.